src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -138,6 +148,28 @@ public enum SecurityCriticalScope {
     Everything = 1,
     Explicit = 0,
   }
+  public sealed partial class SecurityElement {
+    public SecurityElement(string tag) { }
+    public SecurityElement(string tag, string text) { }
+    public System.Collections.Hashtable Attributes { get; set; }
+    public System.Collections.ArrayList Children { get; set; }
+    public string Tag { get; set; }
+    public string Text { get; set; }
+    public void AddAttribute(string name, string value) { }
+    public void AddChild(System.Security.SecurityElement child) { }
+    public string Attribute(string name) { return default(string); }
+    public System.Security.SecurityElement Copy() { return default(System.Security.SecurityElement); }
```

Comment:
In a bunch of these no-op classes we have Copy() return `this` instead of `null`.  Do we want to follow the pattern here?  I expect folks would be very surprised to get `null` back from this method.